mobile/Features/Journal/Components/JournalHistory.jsx

```diff
@@ -0,0 +1,70 @@
+import {
+    ScrollView, Text, View, Button, TextInput, Keyboard, TouchableWithoutFeedback, Modal, TouchableOpacity, Pressable,
+  } from 'react-native';
+  // import PropTypes from 'prop-types';
+  import React, { useState, useEffect } from 'react';
+  import axios from 'axios';
+  import styles from './JournalStyle';
+  import JournalCard from './JournalCard';
+  
+  export default function JournalHistory() {
+
+    const placeholder = [{'username':'nicole','prompt': 'prompt #1', 'text': 'here is my journal post yay!'}, 
+    {'username':'alice','prompt': 'prompt #2', 'text': 'here is my second journal post yay!!'}]
+ 
+    const [journals, setJournals] = useState([]); 
+    const [clickedDate, setClickedDate] = useState(new Date());
+    const [clickedJournal, setClickedJournal] = useState({});
+    const update = (date) => {
+        setClickedDate(date);
+        console.log(clickedDate);
+        //want to get the journal and navigate to journal page
+
+    }
+
+    useEffect(() => {
+        getPastJournals();
+     // setJournals(["hi","hello"])
+      
+      }, []);
+
+    const getPastJournals = async () => {
+      //console.log(`${process.env.EXPO_PUBLIC_SERVER_URL}/journals/getAllJournals`)
+      const res = await axios.get(`${process.env.EXPO_PUBLIC_SERVER_URL}/journals/getAllJournals`);
+      setJournals(res.data);
+    };
+
+    const getJournal = async () => {
+        const res = await axios.get(`${process.env.EXPO_PUBLIC_SERVER_URL}/journals/getAllJournals`);
+        setClickedDate(res.data);
+
+    };
```

Comment:
so sorry queen i'm not understanding this
and the history page did not load properly for me </3